app/assets/images/payment_icons/alchemy.svg

```diff
@@ -0,0 +1 @@
KUnbXnqpLAAAAAAAAAAAAAAAmA5KSUpZv75pkl7vpJOSZNWqwUHJAADAjqNoBgAA ZvIBfZOU8trX1gX4PfeUCAAAAAAAAAAAAAAA08f8+W2bJM97XpIsX758uUwAAGDHUTQDAAAz1MTE uefWR898pjQAAAAAAAAAAAAAAJiO6o1VV69umqTTueyyR7YCAADbm6IZAACYYUZHFywoJWnb17++ bhkYkAoAAAAAAAAAAAAAANNXp5MkbVuLZsbGnvY0mQAAwPanaAYAAGaMurA+f/6LXpQkpey5p0wA AAAAAAAAAAAAAJgpSpk3r5Sk03nTm5JkzZq5c6UCAADbj6IZAACYITZuPOywUpLkvPPqaEEdAAAA AAAAAAAAAICZpZ4Pf/LJnU6yaNE550gEAAC2H0UzAAAwzXW7u+zSNEmn85znlJK07YEHSgUAAAAA AAAAAAAAgJms10tKufTSJOl2V62SCAAAPPUUzQAAwLR31FFtmySnn54kpcyZIxMAAAAAAAAAAAAA AGa6tl2zpmmS5Dd/M0lGR/v7pQIAAE8dRTMAADBNrVu3dGnTJKVcfHEpSSl77SUVAAAAAAAAAAAA AABmg1KSZM6ctk1KOf74JFmw4OijJQMAAE8dRTMAADBNDQysW5ckbXvGGT89fHf8DgAAAAAAAAAA AADArFFKKbVw5rDDmiZp2y1bkuTYY+fPlw4AAGx7LlQFAIBpZvPmxYtLSUp55SuTpJShIakAAAAA AAAAAAAAADD7nXlmkgwPd7t17oatAACwLTnABgCA6XSA3iQPP/zCF9ZG9nXrJAIAAAAAAAAAAAAA wM5j2bJSkra98MIk6XYXLZIJAABsO4pmAABgmuh2Dz88SUp5yUukAQAAAAAAAAAAAADAzqjeuPXk k5OkaU46SSIAALDt9IkAAAB2rG53zpxSklJe//q6ZffdpQIAAAAAALDjjI4ODyfJnDl77pkknU6d T2nbeplDp7N0aZKUsnz54/vK8+Y1TVLKXnslSa83ODhd/sy9XlLKfffVP98tt9Q/V6/3+F5b92/b e+/9ec83zQ9/mCSTk9/6Vt2ydat/ZQAAAADAL1PK0FApSdu+5jVtm3S7H/hAkkxO/vx1SAAA4PFR NAMAADtYKbVpvW3Xrq1zmQAAAAAAAPwy69bVgpeBgQMOePT2Uo4+OknadqrAZXCwFrvst1/bJm27 aNFj9z/ooLr/z26vY9t2Oo/d8lilTG2f2u9XKaVtk16vaeqNCKbPJ0P1go22rY+fWBFM09T9p17/ z792LayZmKjj1F6P/P/cfnvdXsep/55Svv3tXi9p229847Ff8Y476njrrY/ees8911+fJDfe+NBD vksAAAAAYOar65b779+2SSlvfGPdevnlkgEAgCdP0QwAAOwgGzfuvns9dfjFL64nyi5ZIhUAAAAA AGBmqMUqhxxSxyVLBgaS5P77myZJ5s3r76/z+nwpfX1JMjAwMtLpJG27enWS9HrDw22bdDqHHVa/ 7oIFtYRk5cqmSdp2aKjODz20fp2RkcfzX/fz6lvq5zGPf/9ftv3XNR1vPPDYfOrf31P15/rn+y1e XLdP/Tt4RC0Kenxfd/78Oj5SaHPnnfW1t93W6yWl/PCHdT5VXPPgg/VClc98plbe1DshN80ttyTJ Qw/dd1/d7+GH69716/b3/+QnSfLAA/VVDzxQn7/55qmCm59fuAMAAAAAPHGlTFV2X3pp2yYTE3/0 R0nyyU/eeKN0AADgiVM0AwAA29khhwwMNE3S6Tz72XXL6KhUAAAAAACAHaOent/tzp2bJFu3Dg0l SX9/rex4+OHh4brXvHl1/6lx2bJaALJ0aZ2vWpUkw8MLF7Zt0uutXFlKMjhYC0SSPfao45w59c6z dVZraH7ef1Gt6piOhSzMBLXwqJTdd6+VL1M3gPjnpv6d1Vc8si1J+vp+/OM6//a32zYZHr777lrI c+utSTIwcNddvV4yMvKtbyXJxEQd2/ZHP6rfT/X1fX3/9E91ex1/8pNaYDMwUJ+fnHzgAX9nAAAA APCL1cLoqVW817ymbZMNGy65JEmuueaHP5QQAAA8fopmAABgO1u8+MAD6wmrZ5xRT2hdsEAqAAAA AADAttTtThXC7LprkpSy225J0ustWlTnUwUwu+1WT81ftixJmmbRorZN2nb58iTpdJYsqZ9n1Ne3 bR1L6e//Rf/fymGYPWrhUinz5z/23/XatVOPauHSz34PPPhgkvT13X57La353vfqM3fdVUoyMDA1 //a3e71kYuK7363zO+6o36f1wpi2rfs1zV13Jck999x5Z5LceONDD/m7AQAAAGBnU8rUStzGjUnS 6Zx9dp2/+9113LpVSgAA8KspmgEAgO1kzZq5c5smaZqzzqpbjj5aKgAAAAAAwBOxceOSJcmji2D2 3TdJmmblyloYs8cedc8VK2phzMKFdV6LZZpmwYLHFuHvsksd+/qSWpjxy4pilMjA4zE4WMf99qvf M/vt94u+f5omaduHHqqFNN//ft32ox/VZ+++u36//+AHpSQjIz/4QZJ0u7fd1uslyf/9v3W///N/ kqTX+8d/rK//1reSZHLy3nv9XQAAAAAwm9Q1trquXco557Rt0u1efXWSTE5+5SsSAgCAX03RDAAA bCe77LLvvm2blPK859UF7oEBqQAAAAAAAEkyOjo8nCRDQwcdlCT9/cceW4tj9tuvFlAcemiStO3u u9dXzJmTJKXMnVvnU+PQUB0HBn5VaQwwPZTS31+/V5ctq1umxqnnH7t/295/f932wAN1Sy2U6XTu u2/q+STpdr/xjfr55Je+VLd//et1/89+Nkluv/0b30iSW2998EF/CwAAAADMHJ1OkpRy7LFJ0ran nJIkq1ZZ7wIAgMdD0QwAAGwHpSRN89rXlpK0bb2TqBO7AQAAAABg57FhwwEHJElf3zHH1AKZ8fFa IDM6WvdYt
```

Comment:
Also, I see `pi-fps` and FPS as the title. Should we rename to `pi-alchemy` and Alchemy as title?